contract_address.json

```diff
@@ -0,0 +1,135 @@
+[
+    {
+        "Contract Name": "Zone Contract",
+        "Testnet Address": "0x8831867E347AB87FA30199C5B695F0A31604Bb52",
+        "Mainnet Address": "0x00338b92Bec262078B3e49BF12bbEA058916BF91",
+        "Contract Code Link": "",
+        "Tags": ["traders"]
+    },
+    {
+        "Contract Name": "Zone Signer",
```

Comment:
This is not a contract, rather a signer address. Not sure if appropriate here, thoughts @immutable/traders ?